docs/general-concepts/javascript/adding-javascript.md

```diff
@@ -0,0 +1,270 @@
+---
+sidebar_position: 1
+title: Adding JavaScript
+---
+
+Adding JavaScript to your Extension
+===================================
+
+This section covers what you need to do to add JavaScript to your Joomla extension.
+
+## JavaScript Code Files
+
+To add JavaScript files to your extension you should use the Joomla [Web Asset Manager](../web-asset-manager.md) and do the following.
+
+### 1 Store your JavaScript files in the media folder
+
+You should store your JS files in your development media/js folder, for example:
```

Comment:
I think it is would be god to clarify that final location is `media/com_example/js`. Because it can be not very clear from the example.

Addittionaly, when the extension provides some kind of js/css library, let's say `foobar-js`, then the path can be:
`media/com_example/foobar-js/js`

Reply:
I already had the final location noted, but I added a note about implementing a library.